docs/source/modules/zos_script.rst

```diff
@@ -220,8 +220,11 @@ template_parameters
 
     | **required**: False
     | **type**: str
-    | **default**: \\n
-    | **choices**: \\n, \\r, \\r\\n
+    | **default**: 
```

Comment:
Newline options need to also get fixed here

Reply:
regenerated the docs and looks good now